### ISSUE TITLE
[task] research related documents - check the clearmash source data, and find out what we want to show

### ISSUE THREAD

**Issue by OriHoch** [Beit-Hatfutsot/mojp-dbs-pipelines#56]
need to check what data we have regarding related documents in clearmash (e.g. check in the CMS)

then, see if this data is synced by the pipelines

and finally, how we should display it in frontend

related issue: Beit-Hatfutsot/dbs-front#446

**Comment by Libisch** [comment]
@OriHoch Related documents of an item are actually regular items. For example, the related photos of the place item Germany - are _photoUnits_, related places are _place_ type items and so on (these items have their own related documents).

**Comment by Libisch** [comment]
@OriHoch is this still relevant?

**Comment by OriHoch** [comment]
Yes, this is more higher level issue - to check that we show what is needed and that we sync correctly with Clearmash

**Comment by Libisch** [comment]
@OriHoch still relevant?
